MovieViewer/SearchMovieSettings.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SearchMovieSettings.swift
+//  MovieViewer
+//
+//  Created by Dang Quoc Huy on 7/7/16.
+//  Copyright © 2016 Dang Quoc Huy. All rights reserved.
+//
+
+import Foundation
+
+struct SearchMovieParam {
```

Comment:
let me think about it.
I don't see the different.
Furthermore, with enum, in order to get value, i have to add extra .rawValue()